uwblueprint/ccbc
# Change Password Flow

## Notion ticket link

[Change Password Flow](https://www.notion.so/uwblueprintexecs/Change-Password-Flow-d27aed7ac94242ffb7c4f364fbebb49c)


<img width="1792" alt="Screen Shot 2022-03-06 at 1 21 54 PM" src="https://user-images.githubusercontent.com/33358619/156936576-1f2e9c3d-7d4f-4b11-9df5-54da2654fcf1.png">
<img width="1792" alt="Screen Shot 2022-03-04 at 1 09 46 PM" src="https://user-images.githubusercontent.com/33358619/156818492-35fb27ba-c000-49f9-926e-846f4d2f6e59.png">



## Implementation description
* Added a new modal that appears when the Change Password button is pressed in the nav bar
* Component allows the user to change their password by entering their old password and new password



## Steps to test
1. Click on the profile icon and Change Password button 
2. Cases such as entering an incorrect password, different passwords for new and confirm password boxes, etc.


## Checklist
